standard_product_localizer.py

```diff
@@ -54,7 +54,7 @@ def filter(self, record):
 class ACQ(object):
     def __init__(self, acq_id, acq_type):
         self.acq_id=acq_id
-	self.acq_type = acq_type
```

Comment:
changes in this file to resolve this issue when running localizer on Python 3.7.6 on latest hysds isce2 image:
```
Traceback (most recent call last):
  File "initiate_standard_product_localizer.py", line 4, in <module>
    import standard_product_localizer
  File "/home/ops/verdi/ops/standard_product/standard_product_localizer.py", line 52
    self.acq_type = acq_type
                           ^
TabError: inconsistent use of tabs and spaces in indentation
```